### PR TITLE
Update Twin and Methods samples

### PR DESCRIPTION
**Pull request by bikamani**
## Purpose

* Updating sample to show how to unsubscribe for methods and properties

## Does this introduce a breaking change?

```
[ ] Yes
[ ] No
```

## Pull Request Type
What kind of change does this Pull Request introduce?

```
[ ] Bugfix
[ ] Feature
[ ] Code style update (formatting, local variables)
[ ] Refactoring (no functional changes, no api changes)
[ ] Documentation content changes
[ ] Other... Please describe:
```

## How to Test
*  Get the code

```
git clone [repo-address]
cd [repo-name]
git checkout [branch-name]
npm install
```

* Test the code

```
```

## What to Check
Verify that the following are valid
* ...

### CODE REVIEW

File: iot-hub/Samples/device/MethodSample/MethodSample.cs [hunk 1]
```diff
@@ -59,6 +59,17 @@ public async Task RunSampleAsync()
             {
                 await Task.Delay(1000);
             }
+
+            // This is how one can unsubscribe a callback for the methods using a null callback handler
```
Comment (abhipsaMisra) [inline iot-hub/Samples/device/MethodSample/MethodSample.cs:63]: nit: maybe reword: "You can unsubscribe from receiving a callback for direct methods by setting a null callback handler"

**Review by abhipsaMisra — COMMENTED** [review]
Looks good!
You can also update the code snippets here, for easy reference.

**Comment by abhipsaMisra** [comment]
> Looks good!
> You can also update the code snippets here, for easy reference.

Whoops, forgot to link the readme: https://github.com/Azure-Samples/azure-iot-samples-csharp/blob/master/iot-hub/Samples/device/DeviceReconnectionSample/readme.md

**Comment by bikamani** [comment]
> > Looks good!
> > You can also update the code snippets here, for easy reference.
> 
> Whoops, forgot to link the readme: https://github.com/Azure-Samples/azure-iot-samples-csharp/blob/master/iot-hub/Samples/device/DeviceReconnectionSample/readme.md

Do you mean to add code snippet for methods?
